src/components/SocialSharing/SocialSharing.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import SocialSharing from '.';
+
+describe('component SocialSharing', () => {
+  test('should render 4 icons', () => {
```

Comment:
This test says that it checks for 4 rendered icons, but in fact it only checks for one `<img>` tag and one `<svg>` tag. Personally, I would explicitly test for the existence of the 4 sharing buttons. You could use `data-testid` for that: https://testing-library.com/docs/queries/bytestid/